File: 0001 Code Chef/0026 How many unattempted Problem/implementation/0005 kotlin/Unattempted.kt
fun main(){
    var stringInput:String? = readLine();
    if(stringInput != null){
        var stringArray:Array<String>? = stringInput.split(" ").toTypedArray();
        if(stringArray != null && stringArray.size == 2){
            var total:Int? = stringArray[0].toIntOrNull();
            var attempted:Int? = stringArray[1].toIntOrNull();
            if(total != null && attempted != null){
                var unattempted:Int = total - attempted;
                println(unattempted);
            }
        }
    }
}

File: 0001 Code Chef/0026 How many unattempted Problem/implementation/0003 c#/Unattempted.cs
using System;

class Unattempted{
    static void Main(string[] args){
        int total, attempted, unattempted;
        string stringInput = Console.ReadLine();
        string[] stringArray = stringInput.Split(" ");
        total = Convert.ToInt32(stringArray[0]);
        attempted = Convert.ToInt32(stringArray[1]);
        unattempted = total - attempted;
        Console.WriteLine(unattempted);
    }
}